src/index.html

```diff
@@ -5,8 +5,193 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="catalog__header">
+      <div class="header__logo">
+        <a href="/">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__nav">
```

Comment:
```suggestion
      <nav class="nav">
```
отдельный блок, соответственно `nav` -> `nav__list` -> `nav__item` -> `nav__link`

Reply:
ок